scripts/actions/add-files-to-translation-queue.js

```diff
@@ -35,6 +32,29 @@ const slugIntersection = (pendingFiles, filesToRemove) =>
 const humanTranslatedProjectID = process.env.HUMAN_TRANSLATION_PROJECT_ID;
 const machineTranslatedProjectID = process.env.MACHINE_TRANSLATION_PROJECT_ID;
 
+/**
+ * Loads the Exclusions yaml file and converts it to a JSON object.
+ * @returns {Object} The Exclusions yaml file as a JSON object.
+ */
+const getExclusions = () => {
+  return yaml.load(fs.readFileSync(path.join(process.cwd(), EXCLUSIONS_FILE)));
+};
```

Comment:
i think this can just be moved into excludeFiles, or still be kept as a separate method but called directly from excludeFiles.

additionally, i think its cleaner & would allow for us to simplify the signature of excludeFiles, since we wouldn't need to pass `exclusions` parameter anymore.

Reply:
The only trade-off with that is we would be running `yaml.load` and `fs.readFileSync`  for every file, would it not make more sense to only perform that action once?

Reply:
if you run it like this, would it run more that once?:

`const excludeFiles = (fileData) => {
  const exclusions = getExclusions();

  return fileData.filter(({ filename, locale, contentType }) => {
    const localeKey = Object.keys(LOCALE_IDS).find(
      (localeKey) => LOCALE_IDS[localeKey] === locale
    );
    return (
      !exclusions.excludePath[localeKey]?.some((path) =>
        filename.startsWith(path)
      ) && !exclusions.excludeType[localeKey]?.some((type) => contentType === type)
    );
  });
};`

Reply:
ahhh i misunderstood what you meant Kris, that does make complete and utter total sense